app/templates/chat-demo-postgres.html

```diff
@@ -101,15 +101,28 @@ <h1 class="text-center">Chat Demo</h1>
             setupWebscoket(endpoint);
           }
           </script>
-          <script type="module">
-          import { signOut } from '/modules/supabase.js';
-
-            const button = document.getElementById('logout');
-            button.addEventListener('click', async () => {
-            const url = {{ login_url |tojson }};
-            await signOut(url); 
+          <script>
+            document.getElementById('logout').addEventListener('click', async (event) => {
+            console.log("Logging out");
+            event.preventDefault();
+            try {
+                const response = await fetch('/logout', {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({
+                })
```

Comment:
The body params(email and password) are not seen here. Is this somehow connected to the form and will it be taken from there?

Reply:
Yes, I was surprised by that. But `supabase.auth.sign_out()` doesn't take any parameters. I think it must just identify the user from the locally stored accessToken, which we then delete.

Reply:
But how is that? If there are 10 user logged in at a time and we call supabase singout from server side, which user will be signed out.

Removing the token from localstorage may work, but the token also needs to be expired from supabase, right?

Reply:
Hmmm... I've just googled this, and it seems that it isn't easy to log the user out on Supabase from the client side. 

But... does it matter if Supabase doesn't know you have logged out? A user is only logged in if they have a token. If you delete your client-side token, you're not logged in, and nor is anyone else in the world, since they don't have your token either. In order to be logged in, you, or anyone else, has to log in again and receive a new token.

Reply:
I guess it may not matter much. 
But in the event the token is compromised, if the token doesn't get expired from the server, a hacker could access the systems using that token even after the user closes the app or clicks logout. Because from the supabase server side, the token would be still valid. But I dont know how critical an issue this is.

Reply:
Yeah... We could maybe retrieve a new session, and not save the new token? https://supabase.com/docs/reference/python/auth-getsession

Reply:
Looking through the docs makes me more confused. Will supabase allow multiple users to be logged in at the same time? Is it meant to be used from serverside or is it only client side?

Reply:
My impression is that it's designed to be used from client-side. Because I'm realizing, if we want to use other login methods (like Google Account logins, which should be easy), that will only work from client-side. Our current server-side implementation only works if someone is logging in using username and password.